src/intro/mod.rs

```diff
@@ -0,0 +1,92 @@
+extern crate ggez;
```

Comment:
You no longer need `extern crate` in `edition = "2018"` (which is turned on in `Cargo.toml`), except in very specific circumstances.

Reply:
Good point, removed in 11088add95ab99e9238f76c5736e41fc8873c2c4 (chore: removed 'extern crate')

Reply:
+ `cargo fmt` and `cargo clippy` used
+ used ~ syntax in toml